src/app/beer_garden/db/mongo/models.py

```diff
@@ -705,27 +704,6 @@ class CronTrigger(MongoModel, EmbeddedDocument):
     jitter = IntField(required=False)
 
 
-class FileTrigger(MongoModel, EmbeddedDocument):
```

Comment:
This probably needs to stay.  By removing the model you end up with an exception getting thrown by the API if you hit the jobs endpoint (for instance, going to the scheduler page) and a job that uses `FileTrigger` is present.  Even though they never worked, it's possible that someone created a job using that trigger and has it sitting around still.  We don't want to break things in that situation.

Alternatively, we could look to add something to this [`initial_setup`](https://github.com/beer-garden/beer-garden/blob/3.13.0/src/app/beer_garden/db/mongo/api.py#L172) call (probably in `ensure_migrations`) that removes the `FileTrigger` jobs from the database.  Doing that should make it safe to remove this mode.

Reply:
Added a migration to remove the existing file triggers.